From 262e20277bf0bd2b44576ce2bd628c65f5b97305 Mon Sep 17 00:00:00 2001
From: yanghui <353557447@qq.com>
Date: Mon, 31 Oct 2016 18:06:30 +0800
Subject: [PATCH] add num

---
 Student.java | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Student.java b/Student.java
index 4ce1fca..c8e3f71 100644
--- a/Student.java
+++ b/Student.java
@@ -1,3 +1,4 @@
 class Student{
  private String name;
+ int num;
 }
-- 
2.10.1.windows.1

